scripts/scale.sh

```diff
@@ -327,8 +327,11 @@ EOF
 
 	TOTAL_AGENT=$(( $AGENT_NUM + $AGENT_STOCK ))
   for AGENT_I in $( seq $TOTAL_AGENT ); do
-		echo "      - TOR1 -> TOR1_PORT_$AGENT_I" >> $TEMP_DIR/$NAME.yml
-		echo "      - TOR1_PORT_$AGENT_I -> *[Name=agent-$AGENT_I]/eth0" >> $TEMP_DIR/$NAME.yml
+		echo "      - TOR -> TOR_PORT_TOR_$AGENT_I" >> $TEMP_DIR/$NAME.yml
```

Comment:
Is it possible to make it an option of the scale.sh ? We use it a lot for demo and it makes the topology quite complex

Reply:
You want it to be on an argument of the scale.sh script?

Reply:
yes, having this extra TOR topology config optional

Reply:
I just pushed a simpler version still uses "-->".
A feel like a better approach would be to use UT.

Reply:
So you can modify the scale_test.go to add check on the layer2 link you used.